.circleci/images/100-push-images.sh

```diff
@@ -12,6 +13,9 @@ image_tags=(
     "${COLLECTOR_VERSION}-latest"
 )
 for repo in "${image_repos[@]}"; do
+    if [[ "$repo" == "${PUBLIC_REPO}/collector" ]]; then
+        docker login -u "$QUAY_STACKROX_IO_RW_USERNAME" -p "$QUAY_STACKROX_IO_RW_PASSWORD" quay.io
```

Comment:
Perhaps this should done in a a circleci command called earlier in the image job.

Reply:
That cannot happen unfortunately because we are already logged in for `quay.io/rhacs-eng`.

Reply:
Ack.